lua/nvim-tree/actions/tree/modifiers/collapse-all.lua

```diff
@@ -23,21 +23,30 @@ local function buf_match()
   end
 end
 
+---@param node Node
 ---@param keep_buffers boolean
-function M.fn(keep_buffers)
+function M.fn(node, keep_buffers)
```

Comment:
As we are changing the signature of this API, we're going to need to retain backwards compatibility, in the case of the user calling it like `collapse_all(true)`

See https://github.com/nvim-tree/nvim-tree.lua/blob/1ae1c33ce1f4f600dbb9dd8a15b95fbd215358e3/lua/nvim-tree/actions/tree/toggle.lua#L7-L29 for a complex example.